src/components/FormButtons.js

```diff
@@ -32,9 +32,9 @@ import './FormButtons.less';
  */
 
 const FormButtons = ({
-    cancelButtonLabel,
-    saveButtonLabel,
-    childrenPosition,
+    cancelButtonLabel = t`Cancel`,
+    saveButtonLabel = t`Save`,
+    childrenPosition = 'end',
```

Comment:
As https://github.com/bluecatengineering/limani/pull/73/files#r1796537162.

Reply:
I am okay with this change as long as it shows as default value in storybook for this component.